app/recommended-books/release-it/page.css

```
@@ -0,0 +1,557 @@
+    width: 100%;
+    margin: 1.5rem auto 2rem;
+    display: flex;
+    justify-content: center;
+    overflow-x: auto;
+    background: var(--color-pca-s4-surface-1);
+    border: 1px solid var(--color-pca-s4-border);
+    border-radius: var(--radius-pca-s2-card, 14px);
+    padding: 28px 20px;
+}
+
+.release-it-page .mermaid-wrap :global(svg) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)